buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -73,6 +73,8 @@ class ClusterConfiguration {
         return tmpFile.exists()
     }
 
+    Map<String, String> esSettings = new HashMap<>();
```

Comment:
Oddly, I think that the `setting` that you refer to is unused, and what is also puzzling is how it exists yet all the properties that I cut over from using `systemProperty` to `esSetting` were not using it. Either way, I pushed 66ba044ec569f76b6e2b03bfe7ba0912c44707df to cut everything over to `setting`. (I note there is a technical difference between the `esSetting` that I added which the added settings on the command line, and the existing `setting` which added the settings to the config file; however, this difference does not matter.)